cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -242,7 +242,6 @@ rules:
   resources:
   - deployments
   - daemonsets
-  - replicasets
```

Comment:
This likely also needs to be removed from the helm chart and the integration tests (ceph_manifests.go)